exercises/minesweeper/example.py

```diff
@@ -31,9 +33,3 @@ def verify_board(inp):
         cset.update(r)
     if cset - set('+- *|'):
```

Comment:
The second set should be `set(" *")` now, since the other elements were for the border.